### DISCUSSION TITLE
set_chat_menu_button not appearing

### DISCUSSION THREAD

**Discussion by blueprismo** [python-telegram-bot/python-telegram-bot#3938] [Q&A] [answered]
Hello! I am quite new to this framework.
I was trying to set a menu-button like this one: https://core.telegram.org/bots/features#menu-button
```python
async def post_init(application: Application) -> None:
    await application.bot.set_my_commands([('start', 'Starts the bot'), ('help', 'Show some help')])
    await application.bot.set_chat_menu_button()
```
And nevertheless the api calls (https://api.telegram.org/botXXX/setChatMenuButton) and setMyCommands  return`{"ok":true,"result":true}` I can still not see the menu, I am missing something? Thanks in advance!

**Comment by Bibo-Joshi** [comment, marked as answer]
Hi. I recall that my mobile client also tends to cache these settings for quite some time. I recommend trying to restart your Telegram Client, clear the cache, reboot your device or try a different client (e.g. webk.telegram.org).

**Reply by blueprismo** [reply]
That was it... restarting didn't do the trick but trying a different client did it! 
Thanks a lot!

**Comment by Picalz** [comment]


**Reply by Bibo-Joshi** [reply]
Please specify which issues you're facing

**Reply by Picalz** [reply]
Sorry, I figured it out 👍
